pkgs/applications/misc/wayst/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkgconfig
+, freetype
+, fontconfig
+, libGL
+, libX11
+, libXrandr
+, libxcb
+, libxkbcommon
+, utf8proc
+, wayland
+
+, libnotify
+, xdg_utils
+, makeDesktopItem
+, nixosTests
```

Comment:
It doesn't look like `nixosTests` is used.

Reply:
Thanks, I forgot to remove this line. I have tests for this package but I didn't consider them meaningful to add to nixpkgs.